writeengine/splitter/we_sdhandler.cpp

```diff
@@ -1529,7 +1529,8 @@ void WESDHandler::onErrorFile(int PmId, messageqcpp::SBS& Sbs)
     {
         std::string aFile = aTmpFileName.substr(aPos + 1); //+1 to pass '/'
 
-        std::string aInFile = fRef.getLocFile(); // input file
+        std::string aInFile = fRef.fCmdArgs.getErrorDir();
+        //std::string aInFile = fRef.getLocFile(); // input file
```

Comment:
I don't think we need this.

Reply:
This is to make cpimport place all files (log/err/bad) into a specific location. Before some log files would go to /var/lib/columnstore/data/bulk/log and the .bad_[pmID] and .err_[pmID] would go to local directory of input file.

This specific changed line here and WESDHandler::onBadFile make the result .bad .err files from cpimport go to the directory specified by the WECmdArgs directory which is now MCSLOGDIR/cpimport/ by default and can be set by users with cpimport -L option which prior to this was only an option for cpimport.bin

Reply:
I mean we don't need commented row.

Reply:
my mistake.